README.md

```diff
@@ -129,6 +129,7 @@ another database backend that it can query.
 
 Here's a description of Druid from the http://druid.io website:
 
+
```

Comment:
unintended?

Reply:
it was just to allow a non-empty commit to trigger another build... somehow didn't pick up the cache